images/rootfs.yml.in

```diff
@@ -1,9 +1,9 @@
 kernel:
   image: KERNEL_TAG
-  cmdline: "rootdelay=3"
+  # the unified_cgroup_hierarchy forces cgroupsv1, which is required until pillar is ready to support v2
+  cmdline: "rootdelay=3 linuxkit.unified_cgroup_hierarchy=0"
```

Comment:
@rouming will this impact your RT work? I think you talked about cgroupv2.

Reply:
Shouldn't be a problem. Everything now is v1, this makes it explicit. Actually not only does pillar have an issue with v2, runc needs some kernel settings that we don't have enabled. @christoph-zededa knows about them I think. 

Either way, when we are ready, just remove that line and it runs v2. You even can do it interactively at grub time.

Reply:
We do not support v2, so as @deitch said, this makes it explicit. I'm fine with the change.